lib/matplotlib/font_manager.py

```diff
@@ -592,7 +584,7 @@ def createFontList(fontfiles, fontext='ttf'):
                 continue
             try:
                 prop = ttfFontProperty(font)
-            except (KeyError, RuntimeError, ValueError):
+            except (KeyError, RuntimeError, ValueError, NotImplementedError):
```

Comment:
Maybe print to `logging` about it?

Reply:
We already silently drop a bunch of fonts that we don't know how to handle...